fem/config/prometheus.yml

```diff
@@ -6,16 +6,16 @@ global:
 scrape_configs:
   - job_name:       'otlp'
     static_configs:
-      - targets: ['fem_otel-collector:5317']
+      - targets: ['fem-otel-collector:5317']
   - job_name: 'es'
     static_configs:
-      - targets: ['fem_opensearch_exporter:9114']
+      - targets: ['fem-opensearch_exporter:9114']
```

Comment:
Since it is part of the hostname, I suppose the other "_" should also be changed. Do you agree?